jf_agent/validation.py

```diff
@@ -94,9 +94,12 @@ def validate_git(config, creds):
             for repo in git_config.git_include_repos:
                 # Messy: GitLab repos are specified as as ints, not strings
                 if type(repo) == int:
+
                     def comp_func(repo):
                         return repo not in all_repos
+
                 else:
+
```

Comment:
Blank lines here don't really match our convention (maybe this was a merge artifact); I'd just go back to the way this was